src/data/storage.js

```diff
@@ -83,7 +83,9 @@ async function storeDataIntoAkitaFormatNonMonetized(data, typeOfData) {
  * @param {Boolean} isMonetizedData Whether or not the data being stored is for a monetized origin.
  */
 async function updateAkitaData(originData, data, typeOfData, isMonetizedData) {
-	// TODO: ensure typeOfData is one of AKITA_DATA_TYPE
+	if (!Object.values(AKITA_DATA_TYPE).includes(typeOfData)) {
```

Comment:
for readability, maybe we can create something like `const validAkitaDataTypes = Object.values(AKITA_DATA_TYPE)` and then do `validAkitaDataTypes.includes(typeOfData)`